ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -152,7 +152,7 @@ static VALUE rb_mJSON;
 static VALUE rb_cFailedV8Conversion;
 static VALUE rb_cDateTime = Qnil;
 
-static Platform* current_platform = NULL;
+static std::unique_ptr<Platform> current_platform = NULL;
```

Comment:
@SamSaffron, check this out. The new platform API mandates the use of an `sstd::unique_ptr`.

Reply:
oh interesting!